tests/Microsoft.Bot.Builder.LUIS.Tests/LuisRecognizerTests.cs

```diff
@@ -207,9 +209,35 @@ public class LuisRecognizerTests
             Assert.IsTrue((double)result.Entities["Address"][0]["$instance"]["State"][0]["score"] >= 0 && (double)result.Entities["Address"][0]["$instance"]["State"][0]["score"] <= 1);
```

Comment:
This line seems very complex and subtle. 

Can you simplify it?

Reply:
Sure...